src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -209,5 +209,23 @@
 
         <service id="sylius.order_item_names_setter" class="Sylius\Component\Core\Order\OrderItemNamesSetter" />
         <service id="Sylius\Component\Core\Order\OrderItemNamesSetterInterface" alias="sylius.order_item_names_setter" />
+
+        <service id="sylius.last_year_sales_summary_by_postgres_provider" class="Sylius\Component\Core\Provider\LastYearSalesSummaryByPostgresProvider">
+            <argument type="service" id="sylius.repository.order"/>
+            <tag name="sylius.sales_provider" database-platform="postgres" />
+        </service>
+
+        <service id="sylius.last_year_sales_summary_by_mysql_provider" class="Sylius\Component\Core\Provider\LastYearSalesSummaryByMySQLProvider">
+            <argument type="service" id="sylius.repository.order"/>
+            <tag name="sylius.sales_provider" database-platform="mysql" />
+        </service>
+
+        <service id="sylius.last_year_sales_provider" class="Sylius\Component\Core\Provider\LastYearSalesSummaryProvider">
+            <argument type="collection">
+                <argument key="mysql" type="service" id="sylius.last_year_sales_summary_by_mysql_provider" />
+                <argument key="postgres" type="service" id="sylius.last_year_sales_summary_by_postgres_provider" />
+            </argument>
```

Comment:
```suggestion
```
Ref. https://symfony.com/doc/current/service_container/service_subscribers_locators.html#indexing-the-collection-of-services